### PR TITLE
Feature/20097 check submission microservice

### PR DESCRIPTION
**Pull request by danwilliams**

### CODE REVIEW

File: check-submission/src/routes/index.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          resolve(data)
+        } else {
+          reject(new Error('NOT FOUND'))
+        }
+      })
+    })
+  }
+}
+
+// Create the HeroRouter, and export its configured Express.Router
```
Comment (jon-shipley) [inline check-submission/src/routes/index.ts]: HeroRouter?

File: check-submission/src/services/psychometrician-util.service.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+'use strict'
```
Comment (GeorgeChatzigiannis) [inline check-submission/src/services/psychometrician-util.service.ts]: is Ps util service needed for the check submission microservice?

**Comment by GuyHarwood** [comment]
@danwilliams your PRs keep failing the codacy checks (see above).  Can you ensure your editor respects the standardJS policy and this should stop this happening again.

File: check-submission/src/spec/service/psychometrician-util.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
+'use strict'
```
Comment (GeorgeChatzigiannis) [inline check-submission/src/spec/service/psychometrician-util.service.spec.ts]: this file as well

**Comment by GuyHarwood** [comment]
@GeorgeChatzigiannis now that we are consolidating all of the endpoints into one app, i will add a criteria to remove these files and also any other spurious ones.  As Dan is fairly new to the project it will be a lot more time consuming for him to identify these.  I will most likely take care of it once these are merged into master.

**Comment by GeorgeChatzigiannis** [comment]
@GuyHarwood 👍

**Comment by danwilliams** [comment]
@GuyHarwood the issue was that in the change to typescript, the linter was skipping the js files. This is what caused Codacy to complain about that single file. I fixed that gulp file and added standard when linting, for the .js files (at the moment only gulpfile, but maybe others will be added in the future) so the npm lint script now does `standard && tslint --project .`

**Pull request merged**